erela.cli.ts

```diff
@@ -197,11 +202,10 @@ const genreateApp = async () => {
     log(`Created a project ${nameWithPath}`.green.bold)
     const toCp = [
       'package.json',
```

Comment:
This one should be generated and not the same one from the `node_modules` aka the package it self